### ISSUE TITLE
PAR

### ISSUE THREAD

**Issue by hrsdawson** [hrsdawson/ACCESS-WOMBAT_01deg_BGC_validation#10]
@lidefi87 were you interested in looking at this?

**Comment by lidefi87** [comment]
Yes! I could not join this week because I was at the AAPP workshop, but definitely joining next week
